solidity-plugin/src/main/java/org/sonarsource/solidity/CognitiveComplexityVisitor.java

```diff
@@ -118,12 +121,46 @@ private boolean checkExpressionIncrementsComplexity(ExpressionContext exprList)
     return false;
   }
 
+  private static int countConditionalOperators(ExpressionContext expr) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused private "countConditionalOperators" method. ([squid:UnusedPrivateMethod](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3AUnusedPrivateMethod))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=22&issues=AWQ2SgNzYwZU9vvru9og&open=AWQ2SgNzYwZU9vvru9og)

[ISSUE_KEY]: AWQ2SgNzYwZU9vvru9og